libs/partners/mongodb/langchain_mongodb/index.py

```diff
@@ -0,0 +1,105 @@
+import logging
+from typing import Any, Dict, List, Optional
+
+from pymongo.collection import Collection
+from pymongo.operations import SearchIndexModel
+
+logger = logging.getLogger(__file__)
+
+
+def _create_index_definition(
```

Comment:
There is going to be ambiguity here when we add Hybrid Search as we'll have both Vector and Full-Text Indexes. Could we remove please it? It isn't really adding anything. I think that it'll only have one usage, so might as well roll it into `create_vector_search_index`.

Reply:
Maybe rename to `_vector_search_index`?